src/networks/mod.rs

```diff
@@ -27,7 +27,7 @@ const DEFAULT_RECENT_STATE_ROOTS: i64 = 2000;
 
 // Sync the messages for one or many tipsets @ a time
 // Lotus uses a window size of 8: https://github.com/filecoin-project/lotus/blob/c1d22d8b3298fdce573107413729be608e72187d/chain/sync.go#L56
-const DEFAULT_REQUEST_WINDOW: usize = 32;
+const DEFAULT_REQUEST_WINDOW: usize = 8;
```

Comment:
No impact on performances on this?

What motivates this change?

Reply:
We're sending up to 64 requests in parallel, so there's no performance drop in lowering the window size to 8. The standard window size is 8 and some clients disconnect if we set it too high. Better to stay at the standard 8 and make more requests in parallel.